base/src/commonMain/kotlin/text/DateFormatter.kt

```diff
@@ -163,3 +177,51 @@ fun DateFormatter.Companion.fixedPatternFormat(pattern: String, timeZone: TimeZo
  * @param timeZone The [TimeZone] for which the date should be formatted. Defaults to [TimeZone.current].
  */
 fun DateFormatter.Companion.iso8601Pattern(timeZone: TimeZone = TimeZone.current()) = fixedPatternFormat("yyyy-MM-dd'T'HH:mm:ss.SSSZ", timeZone)
+
+/**
+ * Creates a [DateFormatter] that only formats the date components of a [Date]
+ * @param style The [DateFormatStyle] used for formatting the date components of the [Date]. Defaults to [DateFormatStyle.Medium].
+ * @param excludeYear When [true] the year will not be part of the format.
+ * @param timeZone The [TimeZone] for which the date should be formatted. Defaults to [TimeZone.current].
+ * @param locale The [Locale] for which the date should be formatted. Defaults to [Locale.defaultLocale].
+ */
+fun DateFormatter.Companion.dateFormat(
+    style: DateFormatStyle = DateFormatStyle.Medium,
+    excludeYear: Boolean,
```

Comment:
I would make it default to `false`

Reply:
Cant do a default cause it would conflict with the existing dateFormat() method. So there already is a default false in a sense.